.projenrc.ts

```diff
@@ -85,6 +85,9 @@ Closes #<issue number here>.
       printWidth: 120,
       arrowParens: ArrowParens.AVOID,
     },
+    ignoreFileOptions: {
+      ignorePatterns: ['*.md'],
+    },
```

Comment:
We introduced prettier before, but applying it to the current `README.md` would result in a large number of differences.